packages/cardano-services/src/PgBoss/stakePoolBatchDelistHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { CustomError } from 'ts-custom-error';
+import { MissingProgramOption } from '../Program/errors/MissingProgramOption';
+import { POOL_DELIST_SCHEDULE, WorkerHandlerFactory } from './types';
+import { PoolDelistedEntity } from '@cardano-sdk/projection-typeorm';
+import { createSmashStakePoolDelistedService } from '../StakePool/HttpStakePoolMetadata/SmashStakePoolDelistedService';
+
+const createService = (smashUrl: string | undefined) => {
+  if (!smashUrl) throw new MissingProgramOption(POOL_DELIST_SCHEDULE, 'smash-url');
+
+  return createSmashStakePoolDelistedService(smashUrl);
+};
+export const stakePoolBatchDelistHandlerFactory: WorkerHandlerFactory = (options) => {
+  const { logger, smashUrl, dataSource } = options;
+
+  const service = createService(smashUrl);
+  const repo = dataSource.getRepository(PoolDelistedEntity);
+
+  return async (_data: unknown) => {
+    logger.info(`Getting list of delisted pools from ${smashUrl}`);
```

Comment:
In this factory I see 6 `logger` calls. Each contains `stake`, but not this one. 
This one contains only `pools`. It is really low issue, but we can improve it before merging. 

In the end we will have more cohesive logs :)

Reply:
Nooo! I can't let it ruin our logs! I solved it in c4f1525